package.json

```diff
@@ -12,10 +12,10 @@
     "clean": "rm -rf $OUTPUT_DIR",
     "lint": "eslint --fix --ext js --ext jsx src",
     "prod-build-client": "webpack --config ./webpack/config.js",
-    "prod-build-server": "babel ./src -d $OUTPUT_DIR/server --source-maps",
+    "prod-build-server": "babel ./src -d ./build/server --source-maps",
     "prod-build": "npm run clean && npm run prod-build-client && npm run prod-build-server",
     "postinstall": "if [ \"$NODE_ENV\" = production ] ; then npm run prod-build ; fi",
-    "start": "node $OUTPUT_DIR/server/server",
+    "start": "node ./build/server/server",
```

Comment:
This change was already made to the prod instance before I started working on it, so I don't know what the motivation was.